qiita_pet/support_files/doc/source/tutorials/index.rst

```diff
@@ -11,6 +11,7 @@ The following is a full list of the available tutorials:
    getting-started
    analyze-data
    no-raw-sequences
+   portals
```

Comment:
This is not so much a tutorial as it is an admin document, can you create a new folder named admin and move portals.rst along with index.rst? Then in this index.rst document add a new paragraph that reads "The following documents will guide you on how to administer a Qiita portal" ... or something along those lines.